### ISSUE TITLE
[BUG] ImportError in Panel when importing cuxfilter

### ISSUE THREAD

**Issue by mmccarty** [rapidsai/cuxfilter#360]
**Describe the bug**
Nightly package contains a recent [bug in Panel](https://github.com/holoviz/panel/issues/3260), which has been fixed in the latest version 0.12.7.

**Steps/Code to reproduce bug**

`import cuxfilter`

Triggers an ImportError..
```
ImportError: cannot import name 'Markup' from 'jinja2' (/home/mmccarty/miniconda3/envs/rapids-22.04/lib/python3.8/site-packages/jinja2/__init__.py)
```

**Expected behavior**
The import should succeed.

**Environment details (please complete the following information):**
 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)] - All
 - Method of cuxfilter install: [conda, Docker, or from source] - conda & Docker
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used
   - `docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 -v /home/mmccarty/sandbox/rapids/cloud-ml-examples:/rapids/notebooks/cloud-ml-examples rapidsai/rapidsai-core-dev-nightly:22.04-cuda11.5-devel-ubuntu20.04-py3.8`

**Comment by mmccarty** [comment]
cc @ajschmidt8

**Comment by mmccarty** [comment]
`conda install jinja2=2` resolves the `ImportError`. BTW I found this while testing a `cuGraph` notebook. cc @BradReesWork  Thanks @jakirkham

**Comment by jakirkham** [comment]
This should be fixed in `panel` version `0.12.7` via commit ( https://github.com/holoviz/panel/pull/3263/commits/d14ef35b0f9c39d50e6a4d31ecadc1adb8967b60 ). An update to the conda-forge package was added with PR ( https://github.com/conda-forge/panel-feedstock/pull/46 ), but had build issues. Some investigation into that is happening in PR ( https://github.com/conda-forge/panel-feedstock/pull/47 ). Shared some suggestions there.
